core/src/com/unciv/ui/victoryscreen/RankingType.kt

```diff
@@ -0,0 +1,13 @@
+package com.unciv.ui.victoryscreen
+
+enum class RankingType (val value: String) {
+        Population("Population"),
+        CropYield("Crop Yield"),
+        Production("Production"),
+        Gold("Gold"),
+        Land("Land"),
+        Force("Force"),
+        Happiness("Happiness"),
+        Technologies("Technologies"),
+        Culture("Culture")
+    }
```

Comment:
I do like the enum RankingType (though again, I'd rename to CivStatType or something).  And I think this enum might be useful in other parts of the code.  I don't remember where but it was also for stats i think.

Reply:
It is called `RankingType` because it is used for ranking only. That's an enum of ranking aspects.
I don't like naming it as `CivStat` in order to not mess up with existing `Stats` which are used to represent current production, science, etc.